src/Plugin/Field/FieldWidget/GeocoderAutocomplete.php

```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\geocoder_autocomplete\Plugin\Field\FieldWidget\GeocoderAutocomplete.
- */
```

Comment:
phpcs (in my phpstorm) says that is missing file doc comment, if we remove these comments lines.
Why remove?